.gitignore

```diff
@@ -5,4 +5,8 @@ pubspec.lock
 .idea
 .pub/
 .packages
+<<<<<<< HEAD
```

Comment:
Looks like you committed with a merge conflict, maybe?

Reply:
no joel dont you know... thats the cool new .gitignore that will auto solve all merge conflicts.